input/blog/2018-08-28-reactiveui-succession-update.md

```diff
@@ -0,0 +1,19 @@
+---
+title: ReactiveUI Succession Update
+category: 
+  - Announcement
+  - Governance
+  - Succession
+author: Rodney Littles, II
+---
+
+ReactiveUI has released several minor versions since 8.0.  Some of the most recent versions have been released without Geoff having to push code, or prep issues.  This is thanks to members of the core team, and community members who have stepped up to move the project forward.  Glenn Watson has started to play a large role in the maintenance of ReactiveUI.  There have been other's getting involved in enhancing some of the platform logic, and documentation.  Community members have been identifying and resolving issues that help make our framework better.  What does this mean for ReactiveUI succession? It's happening.  What does that mean for you?  It means we are working on ways to improve ReactiveUI and make it easier to consume.  We have several initiatives on the team and there are a few RFC's planned for the 9.0 release that might be of interest.
+
+We are trying to make the contributor experience better.  [RFC 18](https://github.com/reactiveui/rfcs/issues/18) is about enforcing a consistent coding style across the entire code base.  We are going to configure analyzers that will enforce the [Core Fx](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md#c-coding-style) rules.  This will remove the friction of contributors having to check to see if their changes are compliant with our rules before submitting a Pull Request!
```

Comment:
Should you mention the comments being an advantage to the end user btw?

Reply:
Added